### PR TITLE
To-Do-List

### PR DESCRIPTION
**Pull request by sadaf-Daneshgar**
**Add new Features**

- Add Html file
- Add CSS file
- Add JS file
- Configure the webpack
- Configure the linter
- Add all requirements

**Thank you**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,44 +151,26 @@ Feel free to check the [issues page](../../issues/).
...
+I would like to thank Microverse.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```
Comment (TedLivist) [inline README.md:171]: - [ ] Kindly fix your MIT link.

  image

File: README.md [hunk 2]
```diff
@@ -63,174 +32,108 @@ For more information on the importance of a professional README for your reposit
...
+ - Added index.html file.
+ - Added style.css file.
+ - Added javascript file.
+ - Added check errors with linters 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 🚀 Live Demo <a name="live-demo"></a>
 
-<!-- GETTING STARTED -->
+- [Live Demo Link]()
```
Comment (TedLivist) [inline README.md]: - [ ] Kindly deploy the app and provide its link or remove this section on its link on the Table of Contents for a professional README.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @sadaf-Daneshgar,

Good job so far!
✔️ Lists have correct attributes
✔️ Function that renders tasks
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json
Comment (AbelG101) [inline .eslintrc.json:1]: - [ ] [OPTIONAL] The previous reviewer forgot to mention this but please make sure that the name of your branch is the name of a feature or an action. for example `list_structure` or `added_list_structre` would be more appropriate than `first_day` 😉

**Review by AbelG101 — APPROVED** [review]
# Hi @sadaf-Daneshgar 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
- Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json
Comment (sadaf-Daneshgar) [reply .eslintrc.json:1]: Hello @AbelG101 
Ok thank you so much by mention this part I will do!

**Pull request merged**